src/Illuminate/Support/Collection.php

```diff
@@ -296,6 +296,34 @@ public function containsStrict($key, $value = null)
         return in_array($key, $this->items, true);
     }
 
+    /**
+     * Join all items from the collection using a string. The final items can use a separate glue string.
+     *
+     * @param  string  $glue
+     * @param  string  $finalGlue
+     * @return string
+     */
+    public function join($glue, $finalGlue = '')
```

Comment:
IMO, `$finalGlue = null` would be better. Theoretically, someone could actually want to use an empty string.

Reply:
I would also check `func_num_args` instead of comparing it to any value.